DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Reports/ReportsController.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Centre.Reports

```

Comment:
I don't mind too much, but I suspect we won't need a "Reports" folder/can put ReportsController directly into the Centre folder as we don't have too many report related pages.

Reply:
I'm inclined to leave it there right now just to avoid any issues with renaming index methods